spanner/spanner_samples.rb

```diff
@@ -222,6 +222,70 @@ def create_database_with_encryption_key project_id:, instance_id:, database_id:,
   # [END spanner_create_database_with_encryption_key]
 end
 
+def create_database_with_multiple_kms_keys(
+  project_id:, instance_id:, database_id:, kms_key_names:
+)
+  # [START spanner_create_database_with_MR_CMEK]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+  # kms_key_names = "List of Database encryption KMS keys"
```

Comment:
`kms_key_names` is, I understand, a repeated field and should be specified as an array of strings. Please show that so it's clear to someone using the sample how they should set it up. e.g. something like:

```
# kms_key_names = ["Database encryption KMS key 1", "key 2", "key 3"]
```